sdk/eventhub/azure-eventhubs/azure/eventhub/_common.py

```diff
@@ -378,28 +321,27 @@ def try_add(self, event_data):
         :raise: :class:`ValueError`, when exceeding the size limit.
         """
         if event_data is None:
-            log.warning("event_data is None when calling EventDataBatch.try_add. Ignored")
+            _LOGGER.warning("event_data is None when calling EventDataBatch.try_add. Ignored")
             return
         if not isinstance(event_data, EventData):
-            raise TypeError('event_data should be type of EventData')
```

Comment:
Why are we checking the specific type here? Generally, we try to avoid `isinstance` checks...

Reply:
Removed the type checking.